src/IBM.WatsonDeveloperCloud.LanguageTranslator.v3/LanguageTranslatorService.cs

```diff
@@ -286,7 +286,7 @@ public TranslationModel CreateModel(string baseModelId, string name = null, Syst
                     System.Net.Http.Headers.MediaTypeHeaderValue contentType;
                     System.Net.Http.Headers.MediaTypeHeaderValue.TryParse("application/octet-stream", out contentType);
                     forcedGlossaryContent.Headers.ContentType = contentType;
-                    formData.Add(forcedGlossaryContent, "forced_glossary", forcedGlossary.Name);
+                    formData.Add(forcedGlossaryContent, "forced_glossary");
```

Comment:
Also one note here. .NET standard 2 added different overloads for adding to form data. Before this name was null and no error was thrown but now if we try to pass a null value into the overload method that takes a name then an error is thrown. If we don't require a name we need to use the overload that doesn't take a name parameter.

Reply:
Good to know 👍